test/commonjs/error.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const t = require('tap')
+const Fastify = require('fastify')
```

Comment:
(I know these we're capitalised before but it violates the rule of constructor/class pascal case)

```suggestion
const fastify = require('fastify')
```